card/lib/card/mod/dirs.rb

```diff
@@ -72,6 +117,19 @@ def load_from_dir
         end.compact
       end
 
+      def load_from_gemfile
```

Comment:
I think we always have to mix all three options:
gem mods can only be loaded via Gemfiles. Since gems don't have a Gemfile we need the Modfile in the card gem to load the mods of the card gem. If you want to use gem mods in your deck you have to use the Gemfile because otherwise it's not available. Bundler doesn't install mods of the Modfile. For your own local mods you can choose whether you add a Modfile for those in your mod directory or just let decko load everything.